arch/inst/Zkn/aes64ks2.yaml

```diff
@@ -3,9 +3,14 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: aes64ks2
-long_name: No synopsis available
+long_name: This instruction implements part of the KeySchedule operation for the AES Block cipher
```

Comment:
"AES Key Schedule Instruction 2"

...this is from the spec. 32.3.4 "NIST Suite: AES Decryption"